addDevice.py

```diff
@@ -7,19 +7,21 @@
 from werkzeug.security import generate_password_hash
 from argparse import ArgumentParser
 import os
+import configparser
+
 
 parser = ArgumentParser()
 parser.add_argument('-u', '--username', dest='username', required=False)
 parser.add_argument('-p','--password', dest='password', required=False)
 parser.add_argument('-t', '--type', dest='devtype', required=False)
-parser.add_argument('-l', '--list', help='File containing list of users', dest='userlist', required=False)
+parser.add_argument('-i', '--init', help='Init file containing list of users', dest='initfile', required=False)
 args = parser.parse_args()
 # parser.add_argument("Roles", nargs='+')
 
 session = sessionmaker(bind=create_engine(db_path))
 s = session()
 
-
+config_sections = ["device1","device2","device3","device4","device5","device6"]
```

Comment:
Rather then hard coding this, can we iterate over what is in the config file dynamically? That way we can change the section names more easily in the future?